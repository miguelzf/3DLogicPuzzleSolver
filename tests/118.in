- - - - X
- R - - X
- - - - X
L - - - X
- - - - G

- G - - -
- P - - -
- - - - -
X X X - -
L - - - -

- - - - -
- - - P -
- - - - -
X X X X X
- - - - R
