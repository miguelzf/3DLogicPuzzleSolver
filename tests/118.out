
Solution:
g  g  g  g  X  
g  R  r  g  X  
g  g  r  g  X  
L  g  r  g  X  
l  g  r  g  G  
=============
l  G  r  r  r  ||  r  r  r  r  r  
l  P  p  p  p  ||  p  p  p  P  r  
l  l  l  l  r  ||  r  r  r  r  r  
X  X  X  l  r  ||  X  X  X  X  X  
L  l  l  l  r  ||  r  r  r  r  R  
